### PR TITLE
NurseConnect Smsinbound

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps @gsvr please review

File: src/sms_nurse.js [hunk 1]
```diff
@@ -2,21 +2,150 @@ go.app = function() {
...
-                    return "state_start";
+        self.has_active_nurseconnect_subscription = function(id) {
+            return sbm
+            .list_active_subscriptions(id)
+            .then(function(active_subs_response) {
+                var active_subs = active_subs_response.results;
+                for (var i=0; i < active_subs.length; i++) {
+                    // get the subscription messageset
+                    return sbm
+                    .get_messageset(active_subs[i].messageset)
```
Comment (imsickofmaps) [inline src/sms_nurse.js:44]: Promise inside for loop 😢

File: test/sms_nurse.test.js [hunk 2]
```diff
@@ -13,23 +18,114 @@ describe("app", function() {
...
+                        stage_based_messaging: {
+                            url: "http://sbm/api/v1/",
+                            token: "test StageBasedMessaging"
+                        },
+                    },
+                    logging: "off"
                 })
+                .setup(function(api) {
+                    // add fixtures for services used
+                    // fixtures_Hub().forEach(api.http.fixtures.add); // fixtures 0 - 49
```
Comment (imsickofmaps) [inline test/sms_nurse.test.js]: These should not be commented out else the numbering is all off and merge conflicts reign

**Comment by imsickofmaps** [comment]
👍  if the promise loop is handled in follow up

**Pull request merged**